import-automation/executor/gke/configure_gke.sh

```
@@ -14,7 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PROJECT_ID=datcom-data
+# Edit the PROJECT_ID below.
+PROJECT_ID=datcom-import-automation
```

maybe use $1 here and error out if not provided.